x-pack/plugins/canvas/canvas_plugin_src/functions/common/__tests__/axis_config.js

```diff
@@ -59,7 +61,7 @@ describe('axisConfig', () => {
         expect(fn)
           .withArgs(testTable, { position: 'foo' })
           .to.throwException(e => {
```

Comment:
I realize you just followed what was there, but `throwException` can take a RegExp object, so this could also be written as:

```
.to.throwException(new RegExp(functionErrors.axisConfig.positionInvalid('foo').message))
```